src/assets/Assamese.json

```
@@ -1526,591 +1533,583 @@
+  "nurse104Referred": "104 ৰেফাৰ কৰা হৈছে",
+  "presentChiefComplaint": "বৰ্তমান মুখ্য অভিযোগ",
+  "provisionalSelectedDiagnosis": "অস্থায়ী / নিৰ্বাচিত ৰোগ নিৰ্ণয়",
+  "recommendedAction": "পৰামৰ্শিত কাৰ্য",
+  "symptoms": "লক্ষণসমূহ",
+  "symptomResults": "লক্ষণৰ ফলাফল",
+  "diseases": "ৰোগসমূহ",
+  "disease": "ৰোগ",
+  "count": "গণনা",
+  "information": "তথ্য",
+  "dosDont": "কৰিবলগীয়া আৰু নকৰিবলগীয়া",
+  "selfcare": "নিজে যত্ন লোৱা",
```

Biome (2.4.12)

[error] The key selfcare was already declared.

(lint/suspicious/noDuplicateObjectKeys)